.gitignore

```diff
@@ -2,6 +2,8 @@
 .cache
 *.swp
 *~
+*.py
+*.txt
 projects/lambdaman/*.hi
```

Comment:
It probably does not belong to this commit. If you want to keep changes local you can use `.git/info/exclude` file.